search/math/math.md

```diff
@@ -2,6 +2,10 @@
 
 Math modules operate on the pipeline to perform statistical analysis. They are also important when information is condensed over a timeline. For example, if the temperature is measured 10 times per second but the user requests it to be displayed by the second, a math module is used to condense that data.
 
+## Supported Options
```

Comment:
I thought is was for `unique` and `stats` only. I don't think this should be added here.

Reply:
No, it can be used for count, sum, etc. as verified by typing `tag=gravwell syslog Hostname | count -maxtracked 100 by Hostname`

Under the hood, count/sum/mean etc. are all just invoking the stats module anyway.